runner/src/server/views/partials/modal-dialog.html

```diff
@@ -5,8 +5,9 @@ <h1 id="dialog-title" class="heading-large govuk-heading-l">
             Your application will time out soon
         </h1>
         <div class="modal-dialog__inner__text">
-            <div class="timer" aria-hidden="true" aria-relevant="additions"></div>
-            <div id="at-timer" class="at-timer govuk-visually-hidden" role="status"></div>
+<!--            // TODO Uncomment the following two lines to enable session timeout warning -->
+<!--            <div class="timer" aria-hidden="true" aria-relevant="additions"></div>-->
+<!--            <div id="at-timer" class="at-timer govuk-visually-hidden" role="status"></div>-->
```

Comment:
are these ok to remain commented?

Reply:
I've commented out as the timer class element needs to be removed to disable the timer pop as [noted here](https://github.com/communitiesuk/digital-form-builder/blob/5160e09c0e3e3be0ed198d4232371636d318952d/runner/public/static/modal-dialog.js#L21) . I could remove the whole modal-dialog.html file (as the only thing it does is this timer, despite the generic name). But I added it as a TODO and comment as I believe the intention is to disable this for now rather than permanently delete it? Happy to discuss complete removal if that would be better?

Reply:
I could add an env var to disable it with toggle but not sure if that's more functionality than needed in this ticket?